CruzHacks/cruzhacks-2021-website
# Sponsor logos

Problem
=======
As an attendee, I would like to know who is sponsoring the event.
As a sponsor, I need my logo on the website, per the sponsorship agreement.

[Link to Asana Ticket](https://app.asana.com/0/1197738625416321/1199382769817333)

Steps to Verify:
----------------
1. `yarn && yarn start` to initialize development app
2. Check if sponsor URLs lead to expected destinations & component is responsive with devtools

Screenshots (optional):
-----------------------
![Screenshot from 2020-12-30 16-22-58](https://user-images.githubusercontent.com/19658882/103387941-65ab5880-4abb-11eb-978b-b23aa555c058.png)
